contrib/forms/xmlformgen/xslt/report.php.xslt

```diff
@@ -107,7 +107,7 @@ $lists = array(]]></xsl:text>
               $value = $dateparts[0];
             }
 
-	    echo "<td><span class='bold'>";
+	    echo "<td style='width:24%'><span class='bold'>";
```

Comment:
I remember Scott went down this paths in the past but I don't recall the outcome. 2 things to ensure a new pdf library supports are:
1. php7
2. utf8 (non-latin text)